rules/ensure-image-scanning-enabled-cloud/test/failed-no-scan-all/expected.json

```diff
@@ -0,0 +1,54 @@
+[
+   {
+      "fixCommand": "aws ecr put-image-scanning-configuration --repository-name $REPO_NAME --image-scanning-configuration scanOnPush=true --region $REGION_CODE",
+      "alertMessage": "image scanning is not enabled",
+      "failedPaths": [],
+      "fixPaths": [],
+      "ruleStatus": "",
+      "packagename": "armo_builtins",
+      "alertScore": 3,
+      "alertObject": {
+         "externalObjects": {
+            "apiVersion": "eks.amazonaws.com/v1",
+            "data": {
+               "NextToken": null,
+               "Repositories": [
+                  {
+                     "CreatedAt": "2023-01-11T13:07:39Z",
+                     "EncryptionConfiguration": {
+                        "EncryptionType": "XXXXXX",
+                        "KmsKey": null
+                     },
+                     "ImageScanningConfiguration": {
+                        "ScanOnPush": false
+                     },
```

Comment:
Remove this if this test aims to test no "ScanOnPush" configuration a tall...

Reply:
This is a case that should (and does) fail